contrib/copyq/template.py

```diff
@@ -0,0 +1,39 @@
+pkgname = "copyq"
+pkgver = "7.1.0"
+pkgrel = 0
+build_style = "cmake"
+configure_args = [
+    "-DWITH_QT6=ON",
+    "-DCMAKE_BUILD_TYPE=None",
+    "-DCMAKE_INSTALL_PREFIX=/usr",
+]
+hostmakedepends = [
+    "git",
```

Comment:
as far as i can tell git is not needed and whatever data would be used to generate a version would be wrong anyway